rasa/core/test.py

```diff
@@ -667,6 +638,16 @@ def _collect_action_executed_predictions(
                     "training stories and retrain."
                 )
             raise WrongPredictionException(error_msg)
+    elif previously_predicted_action_unlikely_intent:
+        partial_tracker.update(
+            WarningPredictedAction(
+                ACTION_UNLIKELY_INTENT_NAME,
+                predicted_action,
+                prediction.policy_name,
+                prediction.max_confidence,
+                event.timestamp,
+            )
+        )
     else:
         partial_tracker.update(
             ActionExecuted(
```

Comment:
@alwx Don't you also need to transfer the metadata in both of these `elif` / `else` constructs?